Sources/NIOCore/ByteBuffer-views.swift

```diff
@@ -21,8 +21,8 @@ public struct ByteBufferView: RandomAccessCollection, Sendable {
     public typealias Index = Int
     public typealias SubSequence = ByteBufferView
 
-    /* private but usableFromInline */ @usableFromInline var _buffer: ByteBuffer
```

Comment:
@FranzBusch why are we losing all these comments

Reply:
That was on me and I can bring those back. We have a formatting rule that disallows block comments. I removed some of those `private but...` comments since we are super inconsistent with them throughout the project and all the properties are `_`. We have been consistent with the `_` but not so much with the `private but...` comment.